src/SablierV2MerkleLockupFactory.sol

```diff
@@ -1,22 +1,108 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity >=0.8.22;
 
-import { uUNIT } from "@prb/math/src/UD2x18.sol";
 import { ISablierV2LockupLinear } from "@sablier/v2-core/src/interfaces/ISablierV2LockupLinear.sol";
 import { ISablierV2LockupTranched } from "@sablier/v2-core/src/interfaces/ISablierV2LockupTranched.sol";
 import { LockupLinear } from "@sablier/v2-core/src/types/DataTypes.sol";
 
 import { ISablierV2MerkleLL } from "./interfaces/ISablierV2MerkleLL.sol";
 import { ISablierV2MerkleLockupFactory } from "./interfaces/ISablierV2MerkleLockupFactory.sol";
 import { ISablierV2MerkleLT } from "./interfaces/ISablierV2MerkleLT.sol";
-import { Errors } from "./libraries/Errors.sol";
 import { SablierV2MerkleLL } from "./SablierV2MerkleLL.sol";
 import { SablierV2MerkleLT } from "./SablierV2MerkleLT.sol";
 import { MerkleLockup, MerkleLT } from "./types/DataTypes.sol";
 
 /// @title SablierV2MerkleLockupFactory
 /// @notice See the documentation in {ISablierV2MerkleLockupFactory}.
 contract SablierV2MerkleLockupFactory is ISablierV2MerkleLockupFactory {
+    /*//////////////////////////////////////////////////////////////////////////
+                            USER-FACING CONSTANT FUNCTIONS
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @notice inheritdoc ISablierV2MerkleLockupFactory
+    function computeMerkleLL(
```

Comment:
I changed my mind about these utilities. They are verbose, and they might encourage users to prefund the campaigns, whereas our purpose is to get rid of counterfactual usage.

I will remove them.